src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -369,4 +363,38 @@ public ApplianceBootstrapInformationElement getBootstrapinfo(BootStrapInfoProvid
             }
         };
     }
+
+    private DeviceMemberEntity getDeviceMembers(Long deviceId, Long vsMgrId, Long memberId, String memName) {
```

Comment:
getDeviceMember since it just gets one device member

Reply:
OK